### ISSUE TITLE
[1.4.x] Calling the block module gives error

### ISSUE THREAD

**Issue by Portugao** [Guite/MostGenerator#756]
```
Catchable Fatal Error: Argument 1 passed to MU\MUImageModule\Block\Base\ItemListBlock::__construct() must be an instance of MU\MUImageModule\Block\Base\AbstractBundle, instance of MU\MUImageModule\MUMUImageModule given, called in /domain/system/BlocksModule/Api/BlockFactoryApi.php on line 84 and defined
```
 ```
[1] Symfony\Component\Debug\Exception\ContextErrorException: Catchable Fatal Error: Argument 1 passed to MU\MUImageModule\Block\Base\ItemListBlock::__construct() must be an instance of MU\MUImageModule\Block\Base\AbstractBundle, instance of MU\MUImageModule\MUMUImageModule given, called in /www/htdocs/w00b922d/Zikula/Zik143git/system/BlocksModule/Api/BlockFactoryApi.php on line 84 and defined
    at n/a
        in /domain/modules/MU/MUImageModule/Block/Base/ItemListBlock.php line 37

    at Symfony\Component\Debug\ErrorHandler->handleError('4096', 'Argument 1 passed to MU\MUImageModule\Block\Base\ItemListBlock::__construct() must be an instance of MU\MUImageModule\Block\Base\AbstractBundle, instance of MU\MUImageModule\MUMUImageModule given, called in /domain/system/BlocksModule/Api/BlockFactoryApi.php on line 84 and defined', '/domain/modules/MU/MUImageModule/Block/Base/ItemListBlock.php', '37', array())
        in /domain/modules/MU/MUImageModule/Block/Base/ItemListBlock.php line 37

    at MU\MUImageModule\Block\Base\ItemListBlock->__construct(object(MUMUImageModule))
        in /domain/system/BlocksModule/Api/BlockFactoryApi.php line 84

    at Zikula\BlocksModule\Api\BlockFactoryApi->getInstance('MU\MUImageModule\Block\ItemListBlock', object(MUMUImageModule))
        in /domain/system/BlocksModule/Api/BlockApi.php line 135

    at Zikula\BlocksModule\Api\BlockApi->getAvailableBlockTypes(object(ExtensionEntity))
        in /domain/system/BlocksModule/Api/BlockApi.php line 168

    at Zikula\BlocksModule\Api\BlockApi->getModulesContainingBlocks()
        in /domain/system/BlocksModule/Controller/AdminController.php line 70

    at Zikula\BlocksModule\Controller\AdminController->viewAction(object(Zikula_Request_Http))
        in  line 

    at call_user_func_array(array(object(AdminController), 'viewAction'), array(object(Zikula_Request_Http)))
        in /domain/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php line 144

    at Symfony\Component\HttpKernel\HttpKernel->handleRaw(object(Zikula_Request_Http), '1')
        in /domain/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php line 64

    at Symfony\Component\HttpKernel\HttpKernel->handle(object(Zikula_Request_Http), '1', true)
        in /domain/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/DependencyInjection/ContainerAwareHttpKernel.php line 69

    at Symfony\Component\HttpKernel\DependencyInjection\ContainerAwareHttpKernel->handle(object(Zikula_Request_Http), '1', true)
        in /domain/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php line 193

    at Symfony\Component\HttpKernel\Kernel->handle(object(Zikula_Request_Http))
        in /domain/index.php line 23
```

**Comment by Guite** [comment]
@craigh Looks like a core problem?

**Comment by cmfcmf** [comment]
No, this is a MOST issue: "must be an instance of MU\MUImageModule\Block\Base\AbstractBundle". Looks like the import for AbstractBundle is missing.

**Comment by Guite** [comment]
Ah yes, thanks for the pointer!

**Comment by Guite** [comment]
Fixed in 204e5be1ab4ff603ccd1c987b9de71d0674e16ee
